src/app/components/elements/inputs/InlineNumericEntryZone.tsx

```diff
@@ -49,19 +49,24 @@ export const InlineNumericEntryZone = ({width, height, questionDTO, setModified,
         setModified(true);
     }, [value, unit, setModified]);
 
-    return <div {...rest} className={classNames("d-inline-flex inline-numeric-container", rest.className, correctnessClass(valueCorrectness === "NOT_SUBMITTED" ? "NOT_SUBMITTED" : correctness))}>
-        <div className={"feedback-zone inline-nq-feedback"}>
+    return <div {...rest} 
+        className={classNames("inline-numeric-container", rest.className, correctnessClass(valueCorrectness === "NOT_SUBMITTED" ? "NOT_SUBMITTED" : correctness))}
+    >
+        <div className={"feedback-zone inline-nq-feedback"}
+            style={{
+                ...(height && {height: `${height}px`}),
+            }}
+        >
             <Input 
                 ref={focusRef}
                 className={classNames(
-                    "force-print",
+                    "inline-input force-print h-100",
```

Comment:
```suggestion
                    "force-print h-100",
```
`inline-input` no longer exists so can be removed